### PR TITLE
Chore(3.0-beta1): Update core component version to 3.0-beta1

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@harness.io>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: charts/kube-aws/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # kube-aws
 
-![Version: 2.14.0](https://img.shields.io/badge/Version-2.14.0-informational?style=flat-square) ![AppVersion: 2.14.0](https://img.shields.io/badge/AppVersion-2.14.0-informational?style=flat-square)
+![Version: 3.0-beta1](https://img.shields.io/badge/Version-3.0--beta1-informational?style=flat-square) ![AppVersion: 3.0-beta1](https://img.shields.io/badge/AppVersion-3.0--beta1-informational?style=flat-square)
```
Comment (ispeakc0de) [inline charts/kube-aws/README.md:3]:
```suggestion
![Version: 3.0-beta1](https://img.shields.io/badge/Version-3.0-beta1-informational?style=flat-square) ![AppVersion: 3.0-beta1](https://img.shields.io/badge/AppVersion-3.0-beta1-informational?style=flat-square)
```

File: charts/kube-azure/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # kube-azure
 
-![Version: 2.14.0](https://img.shields.io/badge/Version-2.14.0-informational?style=flat-square) ![AppVersion: 2.14.0](https://img.shields.io/badge/AppVersion-2.14.0-informational?style=flat-square)
+![Version: 3.0-beta1](https://img.shields.io/badge/Version-3.0--beta1-informational?style=flat-square) ![AppVersion: 3.0-beta1](https://img.shields.io/badge/AppVersion-3.0--beta1-informational?style=flat-square)
```
Comment (ispeakc0de) [inline charts/kube-azure/README.md:3]:
```suggestion
![Version: 3.0-beta1](https://img.shields.io/badge/Version-3.0-beta1-informational?style=flat-square) ![AppVersion: 3.0-beta1](https://img.shields.io/badge/AppVersion-3.0-beta1-informational?style=flat-square)
```

Comment (ispeakc0de) [reply charts/kube-azure/README.md:3]: check other places too

File: charts/litmus-core/crds/chaosengine_crd.yaml [hunk 3]
```diff
@@ -55,6 +55,44 @@ spec:
                     type: string
                   appns:
                     type: string
+              selectors:
```
Comment (ispeakc0de) [inline charts/litmus-core/crds/chaosengine_crd.yaml:58]: remove  annotationCheck: field

File: charts/litmus-core/crds/chaosengine_crd.yaml [hunk 4]
```diff
@@ -55,6 +55,44 @@ spec:
...
+                        - required: [ names ]
+                        - required: [ labels ]
+                      required:
+                        - kind
+                        - namespace
+                      type: object
+                    type: array
+                oneOf:
+                  - required: [ pods ]
+                  - required: [ workloads ]
```
Comment (ispeakc0de) [inline charts/litmus-core/crds/chaosengine_crd.yaml:95]: add resourceNames in k8sProbe/inputs

**Review by gdsoumya — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

File: charts/kubernetes-chaos/Chart.yaml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "2.14.0"
+appVersion: "3.0-beta1"
 description: A Helm chart to install litmus chaos experiments for kubernetes category (chaos-chart)
 name: kubernetes-chaos
-version: 2.28.0
+version: 2.29.0
```
Comment (Jonsy13) [inline charts/kubernetes-chaos/Chart.yaml:5]: Do we want to make it `3.0-beta1` also?

**Comment by Jasstkn** [comment]
@uditgaurav is this PR still actual?

**Pull request closed without merging**